advanced/index.php

```diff
@@ -209,11 +209,17 @@ function queryAds($serverName) {
     window.onload = function () {
       <?php
       // Remove href fallback from "Back to safety" button
-      if ($featuredTotal > 0) echo '$("#bpBack").removeAttr("href");';
-      // Enable whitelisting if $svPasswd is present & JS is available
-      if (!empty($svPasswd) && $featuredTotal > 0) {
-          echo '$("#bpWLPassword, #bpWhitelist").prop("disabled", false);';
+      if ($featuredTotal > 0) {
```

Comment:
This was a single line statement, why does this now cover the other statement?

Reply:
Now the whitelist button is always enabled (as long as `$featuredTotal > 0`), but the password input is only enabled if there's a password.